front/src/modules/ui/theme/hooks/useColorScheme.ts

```diff
@@ -9,9 +9,12 @@ export function useColorScheme() {
 
   const [updateUser] = useUpdateUserMutation();
 
-  const colorScheme = !currentUser?.settings?.colorScheme
-    ? ColorScheme.System
-    : currentUser.settings.colorScheme;
+  const colorScheme =
+    !currentUser?.workspaceMember?.settings?.colorScheme &&
+    !currentUser?.settings?.colorScheme
+      ? ColorScheme.System
+      : currentUser?.workspaceMember?.settings?.colorScheme ??
+        currentUser.settings.colorScheme;
```

Comment:
Hey @charlesBochet,
Do we need to create an entirely new atom for the workspace-member, or we can read from the `currentUser`?

Reply:
Prisma doesn't permit updating an optional field linked to a model. For instance, while we can update settings via the user (since it's mandatory there), it's not possible through the workspace member due to its optional nature. Currently, our options are to either continue updating settings via the user or to craft a new resolver for direct updates using the `UserSettings` model. We might consider shifting to `WorkspaceMember` once it becomes a mandatory field there. It's worth noting that the other fields we transitioned aren't being updated in the code as of now.

The other fields we've moved don't really get any updates in the code ATM. So, nothing much to do about them other than making copies in WorkspaceMember.

Reply:
We can read from currentUser at this point!